test/gtest/ucm/shm_seg.cc

```diff
@@ -0,0 +1,195 @@
+/**
+* Copyright (C) Mellanox Technologies Ltd. 2001-2015.  ALL RIGHTS RESERVED.
+*
+* See file LICENSE for terms.
+*/
+
+#ifdef HAVE_CONFIG_H
+#  include "config.h"
+#endif
+
+extern "C" {
+#include <ucm/util/sys.h>
+}
+#include <common/test.h>
+
+#include <sstream>
+#include <list>
+
+static const char test_file_01[] = 
```

Comment:
looks like this test is not needed now because the problem is when reading maps file while it's updated ?